analyzers/nodejs/integration_test.go

```diff
@@ -33,9 +32,17 @@ func TestMain(m *testing.M) {
 		panic(err)
 	}
 
-	err = initializeProjects(nodeAnalyzerFixtureDir)
-	if err != nil {
-		panic(err)
+	initializationErrors := fixtures.InitializeProjects(nodeAnalyzerFixtureDir, projects, projectInitializer)
+	initializationErrorFound := false
+	for projName, err := range initializationErrors {
+		if err != nil {
+			initializationErrorFound = true
+			log.Error("Error initializing node project" + projName + " " + err.Error())
+		}
+	}
+
+	if initializationErrorFound {
+		os.Exit(1)
```

Comment:
Why not panic?

Reply:
It makes no difference

Reply:
Panic has better-defined semantics and produces a nicer exit trace. I'd prefer panicking.